src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -416,10 +416,10 @@ private void LogReturnedToken(AuthenticationResult result)
         internal async Task<AuthenticationResult> HandleTokenRefreshErrorAsync(MsalServiceException e, MsalAccessTokenCacheItem cachedAccessTokenItem)
         {
             var logger = AuthenticationRequestParameters.RequestContext.Logger;
-            bool isAadUnavailable = e.IsAadUnavailable();
-            logger.Warning($"Fetching a new AT failed. Is AAD down? {isAadUnavailable}. Is there an AT in the cache that is usable? {cachedAccessTokenItem != null}");
+            
+            logger.Warning($"Fetching a new AT failed. Is exception retry-able? {e.IsRetryable}. Is there an AT in the cache that is usable? {cachedAccessTokenItem != null}");
 
-            if (cachedAccessTokenItem != null && isAadUnavailable)
```

Comment:
Too many terms for the same concept. Better to keep only 1 concept - `is retryable`